test/config/config.go

```diff
@@ -187,6 +189,11 @@ func GetJobsServiceImageTag() string {
 	return env.jobsServiceImageTag
 }
 
+// GetConfigManagementConsoleImageTag return the Kogito Management Console image tag
+func GetConfigManagementConsoleImageTag() string {
```

Comment:
Please change `GetConfigManagementConsoleImageTag` to `GetManagementConsoleImageTag`

Reply:
Fixed.